mlir/include/mlir/Analysis/Presburger/IntegerRelation.h

```diff
@@ -489,11 +489,10 @@ class IntegerRelation {
   void constantFoldVarRange(unsigned pos, unsigned num);
 
   /// Updates the constraints to be the smallest bounding (enclosing) box that
-  /// contains the points of `this` set and that of `other`, with the symbols
-  /// being treated specially. For each of the dimensions, the min of the lower
-  /// bounds (symbolic) and the max of the upper bounds (symbolic) is computed
-  /// to determine such a bounding box. `other` is expected to have the same
-  /// dimensional variables as this constraint system (in the same order).
+  /// contains the points of `this` set and that of `other`. For each of the
```

Comment:
The symbols are still expected to be treated specially. Why is that removed from this comment now? Aren't the users of this method expecting symbolic lower and upper bound functions?